docs/pages/blog/2020.md

```diff
@@ -0,0 +1,115 @@
+---
+title: 2020 in review and beyond
+description: 2019 was a great year for Material-UI. It puts us on an exciting path to solve even greater challenges in the coming years!
+date: 2021-01-01T00:00:00.000Z
+authors: ['oliviertassinari', 'mbrookes']
+card: true
+---
+
+2020 has been another great year not only for Material-UI but also for the ecosystem.
+We have taken on ambitious challenges and started to scale the project.
+
+## Growth
+
+- 📦 From 3.2M to [6.4M](https://npm-stat.com/charts.html?package=%40material-ui%2Fcore&from=2019-11-30&to=2020-12-31) downloads per month (from 14% to 18% of market share in the React ecosystem).
```

Comment:
I have computed it with https://docs.google.com/spreadsheets/d/1l5j3Xjtvm9XZtmb4ulLiWElQaXSlZlyCWT5ONrQMpBo/edit#gid=0, I got 18.6%  which I have floored.

Reply:
Thanks for the link, but I still have no idea what you're measuring since you didn't say, and it's not clear from the npm-stat page what "material-ui" refers to, nor which date nor timeframe you're using.

But taking current weekly downloads of @material-ui/core vs react-dom, it is indeed over 18%.